### PR TITLE
add waterlogging to all cable blocks

### PR DESCRIPTION
**Pull request by Darkere**
implements #2457

### CODE REVIEW

File: src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java [hunk 1]
```diff
@@ -113,6 +119,19 @@ public BlockState getStateForPlacement(BlockItemUseContext ctx) {
         return getState(getDefaultState(), ctx.getWorld(), ctx.getPos());
     }
 
+    public IFluidState getFluidState(BlockState state) {
```
Comment (raoulvdberge) [inline src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java]: `@Override`?

File: src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java [hunk 2]
```diff
@@ -113,6 +119,19 @@ public BlockState getStateForPlacement(BlockItemUseContext ctx) {
...
+        return state.get(WATERLOGGED) ? Fluids.WATER.getStillFluidState(false) : super.getFluidState(state);
+    }
+
+    public boolean receiveFluid(IWorld worldIn, BlockPos pos, BlockState state, IFluidState fluidStateIn) {
```
Comment (raoulvdberge) [inline src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java]: `@Override`?

File: src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java [hunk 3]
```diff
@@ -113,6 +119,19 @@ public BlockState getStateForPlacement(BlockItemUseContext ctx) {
...
+        return IWaterLoggable.super.receiveFluid(worldIn, pos, state, fluidStateIn);
+    }
+
+    public boolean canContainFluid(IBlockReader worldIn, BlockPos pos, BlockState state, Fluid fluidIn) {
```
Comment (raoulvdberge) [inline src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java]: `@Override`?

File: src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java [hunk 4]
```diff
@@ -113,6 +119,19 @@ public BlockState getStateForPlacement(BlockItemUseContext ctx) {
...
+        return IWaterLoggable.super.canContainFluid(worldIn, pos, state, fluidIn);
+    }
+
```
Comment (raoulvdberge) [inline src/main/java/com/raoulvdberge/refinedstorage/block/CableBlock.java]: Needless newline

**Comment by Darkere** [comment]
Copying from vanilla does have it's downsides sometimes

**Comment by raoulvdberge** [comment]
thanks!

**Pull request merged**